### ISSUE TITLE
.noimage/.nomedia doesn't work in Photos gallery

### ISSUE THREAD

**Issue by luciandf** [nextcloud/server#19705]
### Steps to reproduce
1. install latest nextcloud 18.0.1
2. create folder `test`
3. create file `.noimage' or '.nomedia'
4. upload a dir into `test`. the uploaded did contains subdirs with pdf files and image files
5. in the Photos section all the image files in the subdirs appear.

### Expected behaviour
It was my understanding that this was the method to force nextcloud to ignore images in a folder so that they do not appear in the Photos. This method does not work. Expected behaviour was to have no pictures in the Photos section.

### Actual behaviour
All the images from the uploaded dir appear in the Photos sections

**Comment by kesselb** [comment]
Closing as duplicate of https://github.com/nextcloud/photos/issues/75

**Comment by reiner-text** [comment]
@kesselb I can reproduce this behavior with 18.0.1.3, so it cannot be (completely) fixed. If so, then this issue may not be a duplicate. Or did I miss something?

**Comment by kesselb** [comment]
Never the less is https://github.com/nextcloud/server the wrong repository.

**Comment by luciandf** [comment]
it does have some inconsistent behaviour because I notice something when I uploaded the whatsapp folder from my phone with the nextcloud app. it automatically added the .nomedia file and the those folders were not considered in the gallery. then it got me thinking that maybe the app should create these hidden files after the data is uploaded, and yes when i did it that way it worked. 

What I did in the beginning was to create the dir, create the .noimage file, then add the data in that dir.

What I did now, is delete the .noimage file and then re-added it next to the data I uploaded and the unwanted images disappeared from the gallery.

Hope this helps.
